bog_app/app/controllers/creatures_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CreaturesController < ApplicationController
+
+  def index
+    @creatures = Creature.all
+    render :index
+  end
+
+  def new
+    @creature = Creature.new
+    render :new
+  end
+
+  def create
+    creature_params = params.require(:creature).permit(:name, :description)
```

Comment:
Creature params can also be broken out into a private method, which would go underneath the other CRUD actions at the bottom of the file. Then you don't need to redefine creature_params in both your create method and update method, making your code more DRY: 
```
  private

  def creature_params
    params.require(:creature).permit(:name, :description)
  end
```